src/components/RequestsTable/FilterRequests.tsx

```diff
@@ -0,0 +1,433 @@
+                    </Option>
+                  ))}
+                  {GENERALGRADES.map((grade) => (
+                    <Option key={grade} value={grade}>
+                      {grade}
+                    </Option>
+                  ))}
+                </Combobox>
+              )}
+            />
+          </Field>
+          <hr />
+          <Field label="Office Symbol">
+            <Controller
+              name="officeSymbol"
+              control={control}
+              render={({ field }) => <Input type="search" {...field} />}
+            />
+          </Field>
+          <hr />
+          <Field label="Created By">
+            <Input
+              name="Author"
+              defaultValue={filterState
+                .filter((obj) => {
+                  return obj.column === "Author";
+                })[0]
+                ?.filter.toString()}
+            />
+          </Field>
+          <hr />
+          <Field label="Stage">
+            <Controller
+              name="stage"
+              control={control}
+              render={({ field }) => (
+                <Combobox
+                  clearable
+                  autoComplete="on"
+                  {...field}
+                  selectedOptions={[field.value ?? ""]}
+                  onOptionSelect={(_event, data) => {
+                    field.onChange(data.optionValue ?? "");
+                  }}
+                  value={field.value}
+                >
+                  {STAGES.map((stage) => (
+                    <Option key={stage.key} text={stage.text} value={stage.key}>
+                      {stage.text}
+                    </Option>
+                  ))}
+                </Combobox>
+              )}
+            />
+          </Field>
+          <hr />
+          <Field label="Created After">
+            <Controller
+              name="afterDate"
+              control={control}
+              render={({ field }) => (
+                <>
+                  <DatePicker
+                    allowTextInput
+                    formatDate={(date?: Date) => {
+                      return !date ? "" : date.toLocaleDateString();
+                    }}
+                    onSelectDate={field.onChange}
+                    {...field}
+                  />
+                  <Button
+                    appearance="primary"
+                    style={{ marginTop: "2px" }}
+                    onClick={() => field.onChange(undefined)}
+                  >
+                    Clear
+                  </Button>
+                </>
+              )}
+            />
+          </Field>
+          <hr />
+          <Field label="Created Before">
+            <Controller
+              name="beforeDate"
+              control={control}
+              render={({ field }) => (
+                <>
+                  <DatePicker
+                    formatDate={(date?: Date) => {
+                      return date instanceof Date
```

Comment:
Why is this formatDate written differently than the other?  In this case date ?? "" can be replaced with just "", as if it isn't a Date object, then it is undefined so will always just be "" if it reaches it.